cmd/gb/env.go

```diff
@@ -13,8 +13,12 @@ func init() {
 
 var EnvCmd = &cmd.Command{
 	Name:      "env",
-	ShortDesc: "env prints the project environment variables",
-	Run:       env,
+	UsageLine: `env`,
+	Short:     "print project environment variables",
+	Long: `Env prints project environment variables.
+
+`,
+	Run: env,
```

Comment:
Do you mean the extra \n's ? 

Yeah, the conversion to godoc and putting enough \n's to make sure lines are treated as headings requires them. Or at least is very flakey without them.